src/messaging/models.py

```diff
@@ -7,6 +8,11 @@ class Thread(models.Model):
 	profile_1 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_1')
 	profile_2 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_2')
 
+	def getProfileQuery(prof1, prof2):
```

Comment:
I'm actually surprised this works as is.  I've always seen it with the https://docs.python.org/2/library/functions.html#staticmethod.

Sorry for the late comment too, but can you also have it such that if `prof2` isn't supplied to the function, then it will only check if `prof1` is equal to either `profile_1` or `profile_2`?  You can use a default arg like `prof2=None` and then do a `None` check.